lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -744,6 +747,21 @@ class DropDown {
     }
   }
 
+  /**
+   * Focus out handler for the drop down menu.
+   * @param {Event} e
+   */
+  onMenuFocusOut(e) {
+    // The focusout event is not supported in our current version of typescript (3.5.3)
+    // https://github.com/microsoft/TypeScript/issues/30716
+    const focusEvent = /** @type {FocusEvent} */ (e);
```

Comment:
Is there appetite to upgrade typescript?

Reply:
I'm always up for upgrading TS, but we may not be at critical mass yet ... @patrickhulce what do you think?